statesystem/org.eclipse.tracecompass.statesystem.core/src/org/eclipse/tracecompass/internal/statesystem/core/StateSystem.java

```diff
@@ -681,4 +681,19 @@ static void logMissingInterval(int attribute, long timestamp) {
                 attribute + " at time " + timestamp + //$NON-NLS-1$
                 ", returning dummy interval"); //$NON-NLS-1$
     }
+
+    /**
+     * @param t
+     * @param value
+     * @param attributeQuark
+     * @param spanQuark
+     * @throws TimeRangeException
+     * @throws StateValueTypeException
+     */
+    @Override
+    public void modifySpanAttribute(long t, Object value, int attributeQuark, int spanQuark)
```

Comment:
Span is not a state system notion, it's a contextual interval. Should we bring it down to this level of abstraction?